js/src/lib/interfaces/Libraries.ts

```diff
@@ -314,6 +315,38 @@ checkpoint = load_from_hub(
 	filename="{MODEL FILENAME}.zip",
 )`;
 
+const nemoDomainResolver = (domain: string, model: ModelData) => {
+	var model_name = `${(model.id).split("/").pop()}.nemo`;
+
+	switch (domain) {
+		case "ASR":
+		    return `import nemo.collections.asr as nemo_asr
+		    from huggingface_hub import hf_hub_download
+		    
+		    path = hf_hub_download(repo_id="${model.id}",filename="${model_name}")
+		    asr_model = nemo_asr.models.ASRModel.restore_from(path)
```

Comment:
would you be willing to add `huggingface_hub` as a dependency to `nemo` going forward so that the syntax is as simple as possible, e.g.

```python
import nemo.collections.asr as nemo_asr

asr_model = nemo_asr.models.ASRModel.from_pretrained("${model.id}")
```
?

Reply:
Yes that's the plan. A later Nemo point release will have internal support for from_pretrained with huggingface_hub so that users can seamlessly use both Nemo and their own / other public uploaded Nemo models using just one line of from_pretrained

Reply:
Right now we are prototyping this implementation since we are already at code freeze for the current branch, but we expect the point release to be trivial to implement (so soon after the prototype is ready and working, we can finalize the internal operations in Nemo)